### DISCUSSION TITLE
Playbackrate button should accept icons

### DISCUSSION THREAD

**Discussion by luwes** [muxinc/media-chrome#279] [Proposals]
Make it possible to  accept icons and text like the other buttons. 

code here:
https://github.com/luwes/media-theme-neo/blob/main/src/media-playback-rate-range-button.js

example here:
https://media-theme-neo.vercel.app/

**Comment by gkatsev** [comment]
Seems reasonable. Though, I don't think it's great to have icons instead of the values before we get some kind of menu or selector for it.

**Reply by luwes** [reply]
definitely no rush on this. not sure if a menu matters that much, the SVG icons could potentially have text bits too. 
the playback rate button would just be on par with the rest of the buttons.
